subsys/bluetooth/audio/codec.c

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright (c) 2022 Bose Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/** @file
+ *  @brief Bluetooth LE-Audio codec LTV parsing
+ *
+ *  Helper functions to parse codec config data as specified in the Bluetooth assigned numbers for
+ *  Generic Audio.
+ */
+
+#include <bluetooth/audio/audio.h>
+#include <sys/byteorder.h>
+
+/* Overkill for now, but useful if data format is changed from bt_codec_data to TLV format. */
+void bt_codec_parse(const struct bt_codec *codec,
+		   bool (*func)(const struct bt_codec_data *data, void *user_data),
+		   void *user_data)
```

Comment:
Is this function unused?

Reply:
Yes for now, but it allows an application to iterate through the config/capability data rather than calling the getter-functions.